example/PaymentDao.java

```diff
@@ -0,0 +1,31 @@
+public class PaymentDao {
+
+    private static final Logger logger = LoggerFactory.getLogger(PaymentDao.class);
+
+    private final Session session;
+
+    public PaymentDao(Session session) {
+        this.session = session;
+    }
+
+    /**
+     * Saves payment in the database.
```

Comment:
Update the documentation and mention that it throws IllegalArgumentException if payment is null.